src/seedu/addressbook/parser/Parser.java

```diff
@@ -32,7 +29,7 @@
     public static final Pattern PERSON_INDEX_ARGS_FORMAT = Pattern.compile("(?<targetIndex>.+)");
 
     public static final Pattern KEYWORDS_ARGS_FORMAT =
-            Pattern.compile("(?<keywords>\\S+(?:\\s+\\S+)*)"); // one or more keywords separated by whitespace
+                Pattern.compile("(?<keywords>\\S+(?:\\s+\\S+)*)"); // one or more keywords separated by whitespace
```

Comment:
Wrong indentation? [Our coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#layout) requires 4 spaces (instead of tabs) for normal indentation and an indentation of 8 spaces for line wrapping (i.e. when breaking a long line into multiple lines).

Reply:
Furthermore, this change is not related to the enhancement proposed by this PR